python/pyspark/sql/dataframe.py

```diff
@@ -549,6 +549,11 @@ def distinct(self):
     def sample(self, withReplacement, fraction, seed=None):
         """Returns a sampled subset of this :class:`DataFrame`.
 
+        .. note::
```

Comment:
Tiny question about this syntax (I don't know it) -- I see other instances of this in the code base have to use a line continuation `\` when it spans several lines? and then indent continuation lines flush with "..".